build/Analyzers_ShippingRules.ruleset

```diff
@@ -4,7 +4,7 @@
          ToolsVersion="14.0">
 
   <Rules AnalyzerId="Microsoft.ApiDesignGuidelines.Analyzers" RuleNamespace="Microsoft.ApiDesignGuidelines.Analyzers">
-    <!-- DoNotDecreaseInheritedMemberVisibility - blocked by https://github.com/dotnet/roslyn-analyzers/issues/940 -->
+    <!-- DoNotDecreaseInheritedMemberVisibility - blocked by https://github.com/dotnet/roslyn-analyzers/issues/1378 -->
```

Comment:
Have we not published a nuget package since 2222 was removed?

Reply:
We may have, but it's not currently used by this repository. I updated the comment so it's clearer when the restriction is lifted.